bin/eslint-github-init.js

```diff
@@ -68,14 +71,51 @@ inquirer.prompt(questions).then(answers => {
     eslintrc.extends.push('plugin:github/browser')
   }
 
-  if (answers.flow) eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'flow') eslintrc.extends.push('plugin:github/flow')
+  if (answers.typeSystem === 'typescript') {
+    eslintrc.extends.push('plugin:github/typescript')
+
+    // Create a `tsconfig.json` if one doesn't exits. Equivalent to running `tsc --init`.
+    const tsconfigPath = path.resolve(process.cwd(), 'tsconfig.json')
+    if (!fs.existsSync(tsconfigPath)) {
+      const tsconfigDefaults = {
+        compilerOptions: {
+          target: 'es5',
+          module: 'commonjs',
+          strict: true,
+          esModuleInterop: true
+        }
+      }
+      fs.writeFileSync(tsconfigPath, JSON.stringify(tsconfigDefaults, null, '  '), 'utf8')
+    }
+
+    // Create a `tslint.json` if one doesn't exist. Equivalent to running `tslint --init`.
+    const tslintPath = path.resolve(process.cwd(), 'tslint.json')
+    const tslintrc = fs.existsSync(tslintPath)
+      ? JSON.parse(fs.readFileSync(tslintPath, 'utf8'))
+      : {
+          defaultSeverity: 'error',
+          extends: ['tslint-config-prettier'],
+          jsRules: {},
+          rules: {},
+          rulesDirectory: []
+        }
+
+    // If `tslint-config-prettier` isn't the last string in the `extends` array.
+    if (tslintrc.extends[tslintrc.extends.length - 1] !== 'tslint-config-prettier') {
+      tslintrc.extends.push('tslint-config-prettier')
+    }
+    fs.writeFileSync(path.resolve(process.cwd(), 'tslint.json'), JSON.stringify(tslintrc, null, '  '), 'utf8')
```

Comment:
A nice to have here might be to bring in [`detect-indent`](http://npm.im/detect-indent) so that an init doesn't rewrite my files and chose its own indent style.

Reply:
I personally prefer passing in an int instead of a string to `JSON.stringify` as I find it reads clearer:

```suggestion
    fs.writeFileSync(path.resolve(process.cwd(), 'tslint.json'), JSON.stringify(tslintrc, null, 2), 'utf8')
```

Each to their own of course

Reply:
Reading the `detect-indent` API it occurred to me that I can't pass in a number since the indent can be a tab so I pass the `indent` property directly into the JSON stringify function.